src/api/controllers/UserController.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  JsonController, Params, Get, Post, Delete, UseBefore, Body, BodyParam,
+} from 'routing-controllers';
+import { UserModel } from '../../models/UserModel'
+import { UserService } from '../../services/UserService';
+import { Inject } from 'typedi';
+import {
+  GetUsersResponse,
+  GetUserResponse,
+  CreateUserRequest,
+  Uuid,
+  GenericSuccessResponse,
+} from '../../types';
+import { EmailParam, UuidParam } from '../validators/GenericRequests';
```

Comment:
these need to be alphabatized --
import email param,
import Inject needs to come before anything else

Reply:
imports should **always** be alphabetized